File: 241128/ab 사이에 있는 c/c-between-a-and-b.py

arr = list(map(int, input().split()))

a = arr[0]
b = arr[1]
c = arr[2]

satisfied = False # 없다라고 가정 뒤에서 하나라도 있으면 그 때 satisfied == True (YES)로 바꾸기

for i in range(a,b+1) :
    
    if (i%c==0) :
        satisfied = True
    

if satisfied == True :
    print("YES")
else :
    print("NO")

File: 241125/ab 사이에 있는 c/c-between-a-and-b.c
#include <stdio.h>
#include <stdbool.h>

int main() {
    // 여기에 코드를 작성해주세요.
    int a,b,c;
    
    scanf("%d %d %d", &a, &b, &c);
    bool satisfied = false;

    for (int i=a; i<=b; i++){
        if (i%c==0){
            satisfied = true;
        }
    }

    if (satisfied == true){
        printf("YES");
    }
    else {
        printf("NO");
    }

    return 0;
}